.github/container/test-t5x.sh

```diff
@@ -206,3 +234,29 @@ ENABLE_TE=$ENABLE_TE python -m t5x.train \
     $ADDITIONAL_ARGS \
     $([[ $MULTIPROCESS != 0 ]] && echo --multiprocess_gpu)
 echo "Output at ${OUTPUT}"
+
+if [[ "$ENABLE_FMHA" -eq "1" ]]; then 
```

Comment:
Nit: the indenting in this if block is kind of weird. Do you mind normalizing it?